src/app/core/constants/effects.ts

```diff
@@ -0,0 +1,12 @@
+export const buffs = [
```

Comment:
agreed. i tried to use it but apparently there is no way: https://stackoverflow.com/questions/59857223/how-to-convert-typescript-types-of-strings-to-array-of-strings

i need/want an array so that we can iterate the possible values and do ngFor and stuff